commands/device/run-application.ts

```diff
@@ -13,17 +13,14 @@ export class RunApplicationOnDeviceCommand implements ICommand {
 
 	public execute(args: string[]): IFuture<void> {
 		return (() => {
-			this.$devicesServices.initialize({ deviceId: this.$options.device, skipInferPlatform: true }).wait();
+			this.$devicesService.initialize({ deviceId: this.$options.device, skipInferPlatform: true }).wait();
 
-			if (this.$devicesServices.deviceCount > 1) {
-				this.$errors.fail("More than one device found. Specify device explicitly with --device option. " +
-					"To discover device ID, use $%s device command.", this.$staticConfig.CLIENT_NAME.toLowerCase());
+			if (this.$devicesService.deviceCount > 1) {
```

Comment:
There's a missing space after the full stop. Is it possible this message to be shown in the context of Proton? It would be very weird there.